integrations/terraform/envs/platform/providers.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
+  required_version = "~> 1.9.2"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+    conveyor = {
+      source = "datamindedbe/conveyor"
+    }
+  }
+}
+
+provider "aws" {
+  region              = local.aws_region
+  allowed_account_ids = [local.aws_account_id]
+  default_tags {
+    tags = local.mandatory_tags
+  }
+}
+
+# TODO: enable when you set up a kubernetes cluster for datahub and portal or interact directly with conveyor
```

Comment:
Remove unneeded code when you can

Reply:
This code is commented out deliberatly as per the original implementation. Code should or should not be commented out based on the preferred setup of the platform.